src/app/api/label-mapping/label-mapping.service.ts

```diff
@@ -163,4 +163,17 @@ export class LabelMappingService extends BaseService {
       },
     })
   }
+
+  async deleteLabel(label: string) {
+    const currentLabel = await this.db.label.findFirst({
+      where: {
+        label: label,
+      },
+    })
```

Comment:
```suggestion
    const currentLabel = await this.db.label.findFirst({ where: { label } })
```

Reply:
changes had been applied in the commit below